### ISSUE TITLE
member-access rule not working on getters and setters

### ISSUE THREAD

**Issue by lbesson** [palantir/tslint#750]
When applying the member-access rule on the following class
```ts
class Test {
	public static a;
	static b;

	private c;
	d;

	method() {

	}

	private method2() {
	
	}

	get e() { // no error here
		return 2;
	}

	public set f(value) {

	}
}
```
no error is raised for the getter.

Is it the expected behaviour, or should getters and setters also be covered by the rule?

**Comment by adidahiya** [comment]
I believe this was fixed by #801.
